p4-16/spec/P4-16-spec.mdk

```diff
@@ -6941,6 +6941,13 @@ to two controllable entities, which is illegal.
 The `@atomic` annotation, described in Section [#sec-concurrency]
 can be used to enforce the atomic execution of a code block.
 
+### Value set annotations { #sec-value-set-annotations }
+
+The `@match` annotation, described in Section [#sec-select], is used
+to specify a `match_kind` value other than the default `match_kind` of
+`exact` for a field of a `value_set`.
+
+
 ## Package Documentation Annotations { #sec-package-anno }
```

Comment:
Question for others: Should this section have `###` instead of `##`, so that it appears as a sub-section inside of the `## Predefined annotations` section?  Right now it is a sibling in the section hierarchy, not a child, which seems a little off to me.

Reply:
I agree it should be a subsection.

Reply:
I added another commit that moved section 'Package Documentation Annotations' one level lower in the hierarchy.